main/dhcp/dhclient-script-alpine

```diff
@@ -0,0 +1,682 @@
+#!/bin/sh
+# dhclient-script for Linux. Dan Halbert, March, 1997.
+# Updated for Linux 2.[12] by Brian J. Murrell, January 1999.
+# No guarantees about this. I'm a novice at the details of Linux
+# networking.
```

Comment:
While we could maintain a patch, I don't think that would be better. A fork is easier to maintain than a patch.

As for submitting upstream, we can try, but I don't have a lot of hope since ISC has moved their efforts to kea at this point.